File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OcsaOCS0503Q01ListDataInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OcsaOCS0503Q01ListDataInfo
	{
		private String _ioGubun;
		private String _bunho;
		private String _reqDate;
		private String _jinryoPreDate;
		private String _reqGwa;
		private String _reqDoctor;
		private String _appDate;
		private String _consultGwa;
		private String _consultDoctor;
		private String _reqGwaName;
		private String _reqDoctorName;
		private String _consultGwaName;
		private String _consultDoctorName;

		public String IoGubun
		{
			get { return this._ioGubun; }
			set { this._ioGubun = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String ReqDate
		{
			get { return this._reqDate; }
			set { this._reqDate = value; }
		}

		public String JinryoPreDate
		{
			get { return this._jinryoPreDate; }
			set { this._jinryoPreDate = value; }
		}

		public String ReqGwa
		{
			get { return this._reqGwa; }
			set { this._reqGwa = value; }
		}

		public String ReqDoctor
		{
			get { return this._reqDoctor; }
			set { this._reqDoctor = value; }
		}

		public String AppDate
		{
			get { return this._appDate; }
			set { this._appDate = value; }
		}

		public String ConsultGwa
		{
			get { return this._consultGwa; }
			set { this._consultGwa = value; }
		}

		public String ConsultDoctor
		{
			get { return this._consultDoctor; }
			set { this._consultDoctor = value; }
		}

		public String ReqGwaName
		{
			get { return this._reqGwaName; }
			set { this._reqGwaName = value; }
		}

		public String ReqDoctorName
		{
			get { return this._reqDoctorName; }
			set { this._reqDoctorName = value; }
		}

		public String ConsultGwaName
		{
			get { return this._consultGwaName; }
			set { this._consultGwaName = value; }
		}

		public String ConsultDoctorName
		{
			get { return this._consultDoctorName; }
			set { this._consultDoctorName = value; }
		}

		public OcsaOCS0503Q01ListDataInfo() { }

		public OcsaOCS0503Q01ListDataInfo(String ioGubun, String bunho, String reqDate, String jinryoPreDate, String reqGwa, String reqDoctor, String appDate, String consultGwa, String consultDoctor, String reqGwaName, String reqDoctorName, String consultGwaName, String consultDoctorName)
		{
			this._ioGubun = ioGubun;
			this._bunho = bunho;
			this._reqDate = reqDate;
			this._jinryoPreDate = jinryoPreDate;
			this._reqGwa = reqGwa;
			this._reqDoctor = reqDoctor;
			this._appDate = appDate;
			this._consultGwa = consultGwa;
			this._consultDoctor = consultDoctor;
			this._reqGwaName = reqGwaName;
			this._reqDoctorName = reqDoctorName;
			this._consultGwaName = consultGwaName;
			this._consultDoctorName = consultDoctorName;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OcsaOCS0503Q01ListDataInfo.java
package nta.med.data.model.ihis.ocsa;

import java.util.Date;

public class OcsaOCS0503Q01ListDataInfo {
	private String ioGubun;
	private String bunho;
	private Date reqDate;
	private Date jinryoPreDate;
	private String reqGwa;
	private String reqDoctor;
	private Date appDate;
	private String consultGwa;
	private String consultDoctor;
	private String reqGwaName;
	private String reqDoctorName;
	private String consultGwaName;
	private String consultDoctorName;

	public OcsaOCS0503Q01ListDataInfo(String ioGubun, String bunho,
			Date reqDate, Date jinryoPreDate, String reqGwa, String reqDoctor,
			Date appDate, String consultGwa, String consultDoctor,
			String reqGwaName, String reqDoctorName, String consultGwaName,
			String consultDoctorName) {
		super();
		this.ioGubun = ioGubun;
		this.bunho = bunho;
		this.reqDate = reqDate;
		this.jinryoPreDate = jinryoPreDate;
		this.reqGwa = reqGwa;
		this.reqDoctor = reqDoctor;
		this.appDate = appDate;
		this.consultGwa = consultGwa;
		this.consultDoctor = consultDoctor;
		this.reqGwaName = reqGwaName;
		this.reqDoctorName = reqDoctorName;
		this.consultGwaName = consultGwaName;
		this.consultDoctorName = consultDoctorName;
	}

	public String getIoGubun() {
		return ioGubun;
	}

	public void setIoGubun(String ioGubun) {
		this.ioGubun = ioGubun;
	}

	public String getBunho() {
		return bunho;
	}

	public void setBunho(String bunho) {
		this.bunho = bunho;
	}

	public Date getReqDate() {
		return reqDate;
	}

	public void setReqDate(Date reqDate) {
		this.reqDate = reqDate;
	}

	public Date getJinryoPreDate() {
		return jinryoPreDate;
	}

	public void setJinryoPreDate(Date jinryoPreDate) {
		this.jinryoPreDate = jinryoPreDate;
	}

	public String getReqGwa() {
		return reqGwa;
	}

	public void setReqGwa(String reqGwa) {
		this.reqGwa = reqGwa;
	}

	public String getReqDoctor() {
		return reqDoctor;
	}

	public void setReqDoctor(String reqDoctor) {
		this.reqDoctor = reqDoctor;
	}

	public Date getAppDate() {
		return appDate;
	}

	public void setAppDate(Date appDate) {
		this.appDate = appDate;
	}

	public String getConsultGwa() {
		return consultGwa;
	}

	public void setConsultGwa(String consultGwa) {
		this.consultGwa = consultGwa;
	}

	public String getConsultDoctor() {
		return consultDoctor;
	}

	public void setConsultDoctor(String consultDoctor) {
		this.consultDoctor = consultDoctor;
	}

	public String getReqGwaName() {
		return reqGwaName;
	}

	public void setReqGwaName(String reqGwaName) {
		this.reqGwaName = reqGwaName;
	}

	public String getReqDoctorName() {
		return reqDoctorName;
	}

	public void setReqDoctorName(String reqDoctorName) {
		this.reqDoctorName = reqDoctorName;
	}

	public String getConsultGwaName() {
		return consultGwaName;
	}

	public void setConsultGwaName(String consultGwaName) {
		this.consultGwaName = consultGwaName;
	}

	public String getConsultDoctorName() {
		return consultDoctorName;
	}

	public void setConsultDoctorName(String consultDoctorName) {
		this.consultDoctorName = consultDoctorName;
	}

}